tox.ini

```diff
@@ -154,3 +158,9 @@ changedir=packages/bootstrap/extra
 commands=
   pip install .
   pytest --basetemp={envtmpdir} {posargs}
+
+[testenv:mypy]
+deps=
+  mypy==0.650
```

Comment:
Can we use `mypy==0.660`, then we can also add `mypy-mypyc==0.660` and make this faster.